src/test/unit/scheduler/test_compare.py

```diff
@@ -72,22 +72,17 @@ def test_compare_single_run(self):
         compares_done = set()
         self.compare_scheduler.in_queue.put((self.compare_scheduler.db_interface.test_object.uid, False))
         self.compare_scheduler._compare_single_run(compares_done)
-        self.assertEqual(len(compares_done), 1, 'compares done not set correct')
-        self.assertIn(
-            self.compare_scheduler.db_interface.test_object.uid, compares_done, 'correct uid not in compares done'
-        )
+        assert len(compares_done) == 1, 'compares done not set correct'
+        assert self.compare_scheduler.db_interface.test_object.uid in compares_done, 'correct uid not in compares done'
 
     def test_decide_whether_to_process(self):
         compares_done = set('a')
-        self.assertTrue(
-            self.compare_scheduler._comparison_should_start('b', False, compares_done),
-            'none existing should always be done',
-        )
-        self.assertTrue(
-            self.compare_scheduler._comparison_should_start('a', True, compares_done),
-            'redo is true so result should be true',
-        )
-        self.assertFalse(
-            self.compare_scheduler._comparison_should_start('a', False, compares_done),
-            'already done and redo no -> should be false',
-        )
+        assert self.compare_scheduler._comparison_should_start(
+            'b', False, compares_done
+        ), 'none existing should always be done'
```

Comment:
I'm not even sure what the message should tell us ... and it should probably be "non-existing"